pkg/util/common.go

```diff
@@ -0,0 +1,46 @@
+package util
+
+import (
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	pathSeparator = "."
```

Comment:
what's the rationale for using this as a separator vs a normal path separator?

Reply:
This package is used for overlays which use a jsonpath style of path.